docs/api/auth.md

```diff
@@ -0,0 +1,423 @@
+
+## Logging Out
+
+To end your session before the SID expires, you can send a `DELETE` request to the `/api/auth` endpoint. This will invalidate your current SID, requiring you to login again for further requests.
+
+???+ example "Logout"
+
+    === "bash / cURL"
+
+        ``` bash
+        # Example: Logout with SID in the request URI
+        curl -k -X DELETE "https://pi.hole/admin/api/auth?sid=vFA+EP4MQ5JJvJg+3Q2Jnw="
+        ```
+
+    === "Python 3"
+
+        ``` python
+        # Example: Logout with SID in the request header
+        import requests
+
+        url = "https://pi.hole/admin/api/auth"
+        payload = {}
+        headers = {
+          "X-FTL-SID": "vFA+EP4MQ5JJvJg+3Q2Jnw="
+        }
+
+        response = requests.request("DELETE", url, headers=headers, data=payload, verify=False)
+
+        print(response.text)
+        ```
+
+    === "Javascript (plain)"
+
+        ``` javascript
+        var data = null;
+        var xhr = new XMLHttpRequest();
+
+        xhr.addEventListener("readystatechange", function () {
+          if (this.readyState === this.DONE) {
+            console.log(this.responseText);
+          }
+        });
+
+        xhr.open("DELETE", "https://pi.hole/admin/api/auth?sid=vFA+EP4MQ5JJvJg+3Q2Jnw=");
+        xhr.send(data);
+        ```
+
+    === "Javascript (jQuery)"
+
+        ``` javascript
+        $.ajax({
+          url: "https://pi.hole/admin/api/auth",
+          type: "DELETE",
+          data: null,
+          dataType: "json",
+          contentType: "application/json",
+          headers: {
+            "X-FTL-SID": "vFA+EP4MQ5JJvJg+3Q2Jnw="
+          }
+        }).done(function(data) {
+          console.log(data);
+        }).fail(function(xhr, status, error) {
+          console.log(error);
+        });
+        ```
+
+    **Parameters**
+
+    Specify the SID in one of the four ways described above.
+
+???+ success "Success response"
+
+    Response code: `HTTP/1.1 410 - Gone`
+
+    No content
+
+Remember, it's important to manage your sessions properly to maintain the security of your Pi-hole API.
+
+## Rate Limiting
+
+The Pi-hole API implements a rate-limiting mechanism to prevent abuse. This means that you can try a certain number of login attempts per second. If you exceed this limit, you will receive a `429 Too Many Requests` response.
+
+## Limited number of concurrent sessions
+
+The Pi-hole API only allows a limited number of concurrent sessions. This means that if you try to login with a new session while the maximum number of sessions is already active, you may be denied access. This is to prevent abuse and resource exhaustion. In case you hit this limit, please make sure to logout from your sessions when you don't need them anymore as this will free up API slots for future requests. Unused sessions will be automatically terminated after a certain amount of time.
```

Comment:
```suggestion
The Pi-hole API only allows a hard-coded limit of 16 concurrent sessions. This means that if you try to login with a new session while the maximum number of sessions is already active, you may be denied access. This is to prevent abuse and resource exhaustion. In case you hit this limit, please make sure to logout from your sessions when you don't need them anymore as this will free up API slots for future requests. Unused sessions will be automatically terminated after a certain amount of time.
```

Reply:
I did intentionally not specify this here as we may change this in the future or even give users the ability to set the number of maximum concurrent sessions on their own will.